src/common/severity-label/index.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import React from "react";
+import cx from "classnames";
+
+import styles from "./index.css";
+
+const SeverityLabel = ({ severity }) => {
+  const severityClassName = cx("", {
+    [styles.errorSeverity]: severity === "error" || "invalid option",
+    [styles.warningSeverity]: severity === "warning",
+    [styles.severity]: severity !== "warning" && severity !== "error"
+  });
+
+  return <span className={severityClassName}> {severity} </span>;
```

Comment:
```suggestion
  return <span className={severityClassName}>{severity}</span>;
```

Fixes uneven paddings inside label:

<img width="146" alt="Screen Shot 2019-11-16 at 23 36 23" src="https://user-images.githubusercontent.com/654597/69000049-05af1e00-08ca-11ea-9f28-d26f651e36a6.png">

Reply:
Done!